### ISSUE TITLE
Enhancing feed_manager to follow a few models

### ISSUE THREAD

**Issue by a-ta-ta** [GetStream/stream-rails#88]
Using Stream-rails, I noticed that feed_manager allows users follow just other users. 

However, nowadays people follow communities not less than users and it would be awesome to make the  current feed_manager more flexible to use custom models for dealing with feeds.

**Comment by joshmenden** [comment]
I agree with this. I would also like to be able to override the feed that an activity is posted to. It looks like in the source code it is hardcoded to a "user" feed, but what if I want the activity to be posted to a "band" or "artist" feed?

**Comment by joshmenden** [comment]
The above problem of a hard coded "user" is addressed in: https://github.com/GetStream/stream-rails/pull/94
